app/api/api_v1/routers/ingest.py

```diff
@@ -20,60 +22,75 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-def get_collection_template():
+def get_collection_template() -> dict:
+    """
+    Gets a collection template.
+
+    :return dict: The collection template.
+    """
     collection_schema = IngestCollectionDTO.model_json_schema(mode="serialization")
     collection_template = collection_schema["properties"]
 
     return collection_template
 
 
-def get_event_template():
+def get_event_template() -> dict:
+    """
+    Gets an event template.
+
+    :return dict: The event template.
+    """
     event_schema = IngestEventDTO.model_json_schema(mode="serialization")
     event_template = event_schema["properties"]
 
-    del event_template["family_document_import_id"]
-    del event_template["family_import_id"]
-
     return event_template
 
 
-def get_document_template():
+def get_document_template(corpus_type: str) -> dict:
+    """
+    Gets a document template for a given corpus type.
+
+    :return dict: The document template.
+    """
     document_schema = IngestDocumentDTO.model_json_schema(mode="serialization")
     document_template = document_schema["properties"]
-
-    del document_template["family_import_id"]
-    document_template["events"] = [get_event_template()]
+    document_template["metadata"] = get_metadata_template(
+        corpus_type, CountedEntity.Document
+    )
 
     return document_template
 
 
-def get_metadata_template(corpus_type: str):
-    return taxonomy.get(corpus_type)
+def get_metadata_template(corpus_type: str, metadata_type: CountedEntity) -> dict:
+    """
+    Gets a metadata template for a given corpus type and entity.
+
```

Comment:
Missing :param lines for corpus_type and metadata_type - please see [here](https://github.com/climatepolicyradar/navigator-admin-backend/blob/2674536196424175743efe20d72af0cc8cd2942f/app/repository/document.py#L201) for example